tests/languages/php/class-name_feature.test

```diff
@@ -195,6 +197,13 @@ class Foo extends \Package\Bar implements App\Baz {}
 	]],
 	["punctuation", "{"],
 	["punctuation", "}"]
+
+	["keyword", "enum"],
+	["class-name-definition", "Foo"],
+	["keyword", "implements"],
+	["class-name", "Bar"],
+	["punctuation", "{"],
+	["punctuation", "}"],

```

Comment:
The test token stream has to be valid JSON, so this comma is not allowed and there is a comma missing after `["punctuation", "}"]` before `["keyword", "enum"]`.

```suggestion
	["punctuation", "}"]

```

Reply:
Thanks a lot for this. I think was the reason why some tests failed in CI. I force-pushed with this fixed🙏🏼 .